bigflow/__init__.py

```diff
@@ -16,20 +18,27 @@
     'JobContext',
     'Definition',
     'Config',
-    'resources'
 ]
 
-try:
-    from . import bigquery
-    __all__.append('bigquery')
-except ImportError:
-    pass
-
-try:
-    from . import monitoring
-    __all__.append('monitoring')
-except ImportError:
-    pass
+
+# TODO(anjensan): Remove at 1.0
```

Comment:
Why remove at 1.0?

Reply:
It is a hack to provide compatibility with previous versions of `bigflow`, so it might be better to remove it earlier.
We can keep this code for a while however.